src/Cookie.php

```diff
@@ -1,14 +1,35 @@
 <?php
+declare(strict_types=1);
 
 namespace Yiisoft\Yii\Web;
 
+use DateInterval;
+use DateTimeImmutable;
+use DateTimeInterface;
+use Exception;
+use InvalidArgumentException;
 use Psr\Http\Message\ResponseInterface;
 
+use function array_filter;
+use function array_map;
+use function array_shift;
+use function explode;
+use function implode;
+use function in_array;
+use function preg_match;
+use function preg_split;
+use function strtolower;
+
 /**
  * Cookie helps adding Set-Cookie header response in order to set cookie
  */
 final class Cookie
 {
+    // @see https://tools.ietf.org/html/rfc6265#section-4
```

Comment:
Should be phpdoc

Reply:
And it should be `@link` annotation instead of `@see`